docker/mysql_tests/scripts/base_tests/splited_file_stream.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -e -x
+
+. /usr/local/export_common.sh
+
+export WALE_S3_PREFIX=s3://mysqlfullxtrabackupbucket
```

Comment:
Let's use a separate bucket for each test.
Don't forget to create it in `docker-compose.yml`